compress.bash

```diff
@@ -11,19 +11,19 @@
 # Name of the ROM. No Spaces Please.
 # Example: CyanogenMod
 
-name=ColorOS
+name=AOSP-RRO
 
 
 # Manifest link. https:// is mandatory.
 # Example: https://github.com/cyanogenmod/android
 
-manifest=https://github.com/Qiangong2/patchrom/default.xml
+manifest=https://github.com/AOSP-RRO/manifest -b nougat
```

Comment:
the branch is nougat, -b nougat is the branch.
https://github.com/AOSP-RRO/manifest